spec/requests/evaluations_spec.rb

```diff
@@ -63,4 +63,84 @@
       end
     end
   end
+
+  describe "GET /evaluations/:id/submissions" do
+    let(:evaluator) { create(:user, role: 'evaluator') }
+    let(:challenge) { create(:challenge) }
+    let(:phase) { create(:phase, challenge: challenge) }
+    let!(:evaluation_form) { create(:evaluation_form, phase: phase, challenge: challenge) }
+
+    context "when logged in as an evaluator" do
```

Comment:
it might be good to have a context `when logged in as an evaluator that is not associated with the challenge phase` to assert access to unrelated phases is denied?